### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by tprelog**
Give this a try -- It's the same... just different

I've never used Movie Night before so I'm not sure what to expect.

After installation, I reach this screen by pressing the MANAGE button in TrueNAS UI

image

### CODE REVIEW

**Comment by tprelog** [comment]
By the way, these are the only packages I have my in plugin manifest
```
    "pkgs": [
        "bash",
        "ca_root_nss",
        "git-lite",
        "go"
    ],
```

I can also push the changes (I made in the pull request) to the master branch on my fork, if it makes it easier for you to test

**Comment by zorglube** [comment]
God I have so much to learn ^_^

**Comment by tprelog** [comment]
@zorglube hang on -- your test did not work because I have those changes in a separate branch...

Give me on minute, i was merge changes to my master branch so you can test

**Comment by tprelog** [comment]
@zorglube if you test again. it will work this time using my repo

File: post_install.sh [hunk 1]
```diff
@@ -1,11 +1,26 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-chmod +x /usr/local/movienight/MovieNight
+## Who will run the jail's primary service
+service_port="8089"        # service_port == UID
```
Comment (zorglube) [inline post_install.sh:4]: Why using the listening port as UID ?

**Review by zorglube — COMMENTED** [review]
> @zorglube hang on -- your test did not work because I have those changes in a separate branch...

I'll keep on learning ;-) 
It worked fine. 

> Give me on minute, i was merge changes to my master branch so you can test

I'll certainly merge you PR.

**Comment by zorglube** [comment]
> this

It is !

File: post_install.sh (hunk 1 above)
Comment (tprelog) [reply post_install.sh:4]: Typically, the Username is similar to the name of the primary service and the User ID usually matches the default port being used.

Comment (zorglube) [reply post_install.sh:4]: Ok, but, does it have any technical interest or is this just a way to remind what is the listening port of the jail ?

**Comment by zorglube** [comment]
> Give this a try -- It's the same... just different 

The same-but-different ^_^ so are you able to explain me why my-same (^_^) didn't worked ? 
 
> I've never used Movie Night before so I'm not sure what to expect.

Exactly what expected !

File: post_install.sh (hunk 1 above)
Comment (tprelog) [reply post_install.sh:4]: It's just a _suggested_ good practice... I read it about in the old FreeNAS documentation awhile back. I'll share the link if I can find it it. I follow this practice for all of my plugins bur it is certainly not required. Feel free to change it if you want

Comment (zorglube) [reply post_install.sh:4]: Ok, that's what I has assumed. Thank you ! Thank a lot for your precious help !

Comment (tprelog) [reply post_install.sh:4]: I'm glad I could help :slightly_smiling_face:

**Comment by tprelog** [comment]
> God I have so much to learn ^_^

Well keep at it, you get the hang of it.

For help understanding `/usr/etc/rc.d/movienight` - this is a great guide
- https://docs.freebsd.org/en_US.ISO8859-1/articles/rc-scripting/index.html

> The same-but-different ^_^ so are you able to explain me why my-same (^_^) didn't worked ?

I can't really say for sure - I changed everything at once. Keep in mind, I started making plugins on FreeNAS 11.2, when they were available via CLI install only - so I have some practice.

**Comment by zorglube** [comment]
Thank for the doc ;-)

**Pull request merged**